<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogGui</class>
 <widget class="QWidget" name="LogGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget_log">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QLineEdit" name="lineEdit_numOfAttempts">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>210</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_reaction">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>320</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_name">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>30</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_age">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>90</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Age (Must be 18 or older)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_enter">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>480</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(30,144,255);
color:rgb(255,255,255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;

</string>
       </property>
       <property name="text">
        <string>Enter</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_committed">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>420</y>
         <width>241</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Commit to volunteering? (true or false)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_date">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>370</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Date (mm/dd/yyyy)</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_name">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>30</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_reaction">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>320</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Reaction to the interaction</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_phoneNumber">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>150</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Phone Number (XXXXXXXXXX)</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_methodOfContact">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>260</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_numOfAttempts">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>210</y>
         <width>181</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Number of outreach attempts</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_committed">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>420</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_phoneNumber">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>150</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_age">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>90</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_date">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>370</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_methodOfContact">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>260</y>
         <width>241</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Method of contact (eg. text, phone call)</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_name</tabstop>
  <tabstop>lineEdit_age</tabstop>
  <tabstop>lineEdit_phoneNumber</tabstop>
  <tabstop>lineEdit_numOfAttempts</tabstop>
  <tabstop>lineEdit_methodOfContact</tabstop>
  <tabstop>lineEdit_reaction</tabstop>
  <tabstop>lineEdit_date</tabstop>
  <tabstop>lineEdit_committed</tabstop>
  <tabstop>pushButton_enter</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
